packages/plugin-conversation/src/share-activity.js

```diff
@@ -255,10 +270,26 @@ const ShareActivity = SparkPlugin.extend({
 
   /**
    * @param {Array} items
+   * @param {string} mimeType
+   * @private
+   * @returns {boolean}
+   */
+  _itemContainsActionWithMimeType(items, mimeType) {
+    return some(items.map((item) => some(item.actions, [`mimeType`, mimeType])));
```

Comment:
the rest of the sdk uses object instead of array notation for lodash predicates. can you changes this to `{mimeType: mimeType}`?